openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -1448,3 +1376,254 @@ def update_plot(ii):
     plt.show()
     return line_ani
 
+
+
+
+
+# =============================================================================
+# Experimental Functions
```

Comment:
I like the idea of plot on click. I would want to keep it permanently. However, let's keep this "Experimental Functions" heading for now so we know this is something we should discuss a lot.

Reply:
Okay, good to know!